Settings.XamlStyler

```diff
@@ -0,0 +1,41 @@
+{

```

Comment:
Make sure you add this file to consolegit2gitfilters

Reply:
good point